addons/mail/controllers/message_reaction.py

```diff
@@ -5,13 +5,20 @@
 from odoo import http
 from odoo.http import request
 from odoo.addons.mail.models.discuss.mail_guest import add_guest_to_context
+from odoo.addons.mail.tools.discuss import Store
 
 
 class MessageReactionController(http.Controller):
     @http.route("/mail/message/reaction", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
-    def mail_message_add_reaction(self, message_id, content, action):
+    def mail_message_reaction(self, message_id, content, action, **kwargs):
         message = request.env["mail.message"].browse(int(message_id)).exists()
         if not message._validate_access_for_current_persona("write"):
```

Comment:
@seb-odoo Do you think we need to change it to `create` here as well?

Reply:
Yes I think with `write` it will fail for threads with post access as `read` for users with only `read` access on thread (if `_get_mail_message_access` isn't also overridden, which is often not the case).

But I'll change it in my PR/forward-port, with additional tests.